presto-docs/src/main/sphinx/functions/math.rst

```diff
@@ -79,6 +85,13 @@ Mathematical Functions
     The successProbability and p must be real values in [0, 1] and the numberOfTrials must be
     a positive integer.
 
+.. function:: inverse_cauchy_cdf(median, scale, p) -> double
+
+    Compute the inverse of the Cauchy cdf with given parameters median and
+    scale (gamma) for the probability p.
+    The scale parameter must be a positive real number.
```

Comment:
use "real value" or double, or mention real number of type double. `REAL` is a type so `real number` is a bit ambiguous.